resources/profiles/Anker/process/0.15mm Fast @Anker.json

```diff
@@ -0,0 +1,42 @@
+{
+    "type": "process",
+    "setting_id": "GP_Anker_002",
+    "name": "0.15mm Fast @Anker",
+    "from": "system",
+    "instantiation": "true",
+    "inherits": "fdm_process_common",
+    "bottom_shell_layers": "5",
+    "enable_overhang_speed": "1",
+    "bridge_flow": "0.85",
+    "initial_layer_print_height": "0.15",
+    "layer_height": "0.15",
+    "raft_contact_distance": "0.15",
+    "top_shell_layers": "6",
+    "support_bottom_z_distance": "0.15",
+    "support_top_z_distance": "0.15",
+    "top_surface_pattern": "rectilinear",
+    "bottom_surface_pattern": "rectilinear",
```

Comment:
it should be zig-zag，Rectilinear  is a label ,the real value  is zig-zag

Reply:
changed as requested

Reply:
There are still some existing rectilinear patterns, and these values will cause the preset to fail to load properly. Could you please make the necessary modifications ?

Reply:
@XunZhangBambu, there should be no more rectilinear values present now. One thing to note is there are A LOT of other profiles that use rectilinear.